tests/ui/parallel-rustc/cache-after-waiting-issue-111528.stderr

```diff
@@ -1,5 +1,5 @@
 error: symbol `fail` is already defined
-  --> $DIR/cache-after-waiting-issue-111528.rs:12:1
+  --> $DIR/cache-after-waiting-issue-111528.rs:13:1
```

Comment:
Question: something else that came to my mind is that are these line numbers reproducible if ran from different number of threads? Do they ever need to be normalized into like `:LL:COL`?

Reply:
Thanks for reminding me, I deleted all the error output information, which will greatly simplify this PR

Reply:
Though if we no longer have the `-Z threads=1`, would *other* non-parallel tests potentially become flaky?